snuba/datasets/schema.py

```diff
@@ -72,15 +72,19 @@ def __init__(self, local_table_name, dist_table_name, columns,
         self.__sample_expr = sample_expr
 
     def _get_local_engine(self):
-        partition_by_clause = "PARTITION BY %s" % \
-            self.__partition_by if self.__partition_by else ''
+        partition_by_clause = ("PARTITION BY %s" %
+            self.__partition_by) if self.__partition_by else ''
+
+        sample_clause = ("SAMPLE BY %s" %
+            self.__sample_expr) if self.__sample_expr else ''
+
         return """
             ReplacingMergeTree(%(version_column)s)
              %(partition_by_clause)s
             ORDER BY %(order_by)s
-            SAMPLE BY %(sample_expr)s ;""" % {
+             %(sample_expr)s ;""" % {
```

Comment:
```suggestion
             %(sample_clause)s ;""" % {
```

This is no longer an expression.